contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -233,6 +229,13 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint32 callbackGasLimit,
     uint256 gasPrice
   ) external view override returns (uint96) {
+    if (callbackGasLimit > s_config.maxCallbackGasLimit) {
```

Comment:
This will be addressed by https://github.com/smartcontractkit/chainlink/pull/9903